Automattic/pocket-casts-ios
# Adding a Share extension for OPML exports

currently, even though we already have the code that handles OPML files, but the app doesn't appear as an option in the share sheet and I have to scroll to More and then search Pocket Casts among many other apps

https://github.com/Automattic/pocket-casts-ios/assets/1335657/876c97c7-6bbf-4abe-a47b-eb8c99cfff89

This PR adds a share extension so that the app shows up on OPML shares (exports). 
Please note that I left the boilerplate code in `ShareExtensionViewController` as our app handles opml files in `handleOpenUrl`

With this change:

https://github.com/Automattic/pocket-casts-ios/assets/1335657/8323f7bc-3dde-43c3-8cea-4384eca1014f

## To test
Steps to test your PR.
Export OPML files from other apps and see Pocket Casts as an option in the apps row of the share sheet.

## Checklist
